mirage/serializers/application.js

```diff
@@ -47,7 +47,7 @@ export default Serializer.extend({
   },
 
   keyForEmbeddedRelationship(attributeName) {
-    if(attributeName === 'subjects' || attributeName === 'customerResources') {
+    if(attributeName === 'customerResources' || attributeName === 'subjects' || attributeName === 'contributors') {
```

Comment:
i prefer your approach in https://github.com/thefrontside/ui-eholdings/pull/51 to this one

Reply:
Second